diff --git a/statick_tool/statick.py b/statick_tool/statick.py
index ed5f921..a6b4e20 100644
--- a/statick_tool/statick.py
+++ b/statick_tool/statick.py
@@ -469,13 +469,18 @@ class Statick:
         total_issues = []
         num_packages = len(packages)
         mp_args = []
-        for package in packages:
-            count += 1
-            mp_args.append((parsed_args, count, package, num_packages))
-
         print("-- Scanning {} packages --".format(num_packages), flush=True)
-        with multiprocessing.Pool(parsed_args.max_procs) as pool:
-            total_issues = pool.starmap(self.scan_package, mp_args)
+        if multiprocessing.get_start_method() == "fork":
+            for package in packages:
+                count += 1
+                mp_args.append((parsed_args, count, package, num_packages))
+
+            with multiprocessing.Pool(parsed_args.max_procs) as pool:
+                total_issues = pool.starmap(self.scan_package, mp_args)
+        else:
+            for package in packages:
+                count += 1
+                self.scan_package(parsed_args, count, package, num_packages)
 
         print("-- All packages run --")
         print("-- overall report --")
